pcdsdevices/ccm.py

```diff
@@ -903,11 +938,20 @@ class CCM(BaseInterface, GroupDevice, LightpathMixin, CCMConstantsMixin):
     energy_with_vernier = Cpt(
         CCMEnergyWithVernier, '', kind='normal',
         doc=(
-            'PsuedoPositioner that moves the alio in '
+            'PseudoPositioner that moves the alio in '
             'terms of the calculated CCM energy while '
             'also requesting a vernier move.'
         ),
     )
+    energy_with_acr_status = Cpt(
+        CCMEnergyWithACRStatus, '', kind='normal',
+        doc=(
+            'PseudoPositioner that moves the alio in '
+            'terms of the calculated CCM energy while '
+            'also requesting an energy change to ACR. '
+            'This will wait on ACR to complete the move.'
```

Comment:
So as far as I can see, these two energy components are identical except one sends and waits on a request to ACR.  Is there ever a case where we wouldn't want to send a request to ACR?  If ACR is already at the setpoint, is there an issue with sending the request anyway?

I'm mostly asking since having two components might be confusing down the line.  The docs are nice and clear, I'm just wondering if we can remove opportunities for mistakes.

Reply:
Yes there is. When we do fast energy scans over small ranges, we do a pseudo fly scan, where we don't need / want to wait on ACR. When doing larger ranges, when the K needs to move, things are much slower and we need to wait.